README.md

```diff
@@ -90,15 +92,6 @@ Versions are auto incremented based on commit messages and SemVer (Major.Minor.P
 - `patch:` or `fix:` example `fix: url bug` 
   - this will increment the PATCH version eg 1.0.0 to 1.0.1. This will also happen by default.
 
-#### Build process
-
-1. Pushes to any branch will start the build process
-2. `.circleci/circle_trigger.sh` will check for any changes in our packages, and if builds have failed previously
-3. `circle_trigger.sh` will trigger a workflow via the API. It will pass the parameters model, runner, designer (bool) to the workflow.
-4. If there are any changes to a workspace, it will be built and tested.
-
-- If an upstream dependency, like model has changed, the downstream dependencies (runner, designer) will also be built and tested.
```

Comment:
Hi. We need to change this to active voice - please add who or what builds and tests the downstream dependendies. Please also code out model, runner and designer if appropriate, and change to the following (and remove the bullet before ' - ').
For example:

If an upstream dependency, such as `model`, has changed, xx-THE-WHAT-OR-WHO-xx will also build and test the downstream dependencies (`runner`, `designer`).